Buildings/Controls/OBC/FDE/PackagedRTUs/OperatingMode.mo

```diff
@@ -0,0 +1,264 @@
+                                                  lineColor={179,151,128},
+          radius=20,
+          fillColor={255,255,255},
+            fillPattern=FillPattern.Solid),
+        Text(
+          extent={{-54,150},{68,94}},
+          lineColor={28,108,200},
+          fillColor={179,151,128},
+          fillPattern=FillPattern.Solid,
+          textString="%name"),
+        Text(
+          extent={{-98,66},{-72,52}},
+          lineColor={217,67,180},
+          textString="Occ"),
+        Text(
+          extent={{-96,36},{-70,22}},
+          lineColor={0,0,255},
+          textString="OccReq"),
+        Text(
+          extent={{-96,6},{-72,-8}},
+          lineColor={0,0,255},
+          textString="TotalTU"),
+        Text(
+          extent={{-96,-24},{-72,-38}},
+          lineColor={0,0,255},
+          textString="SBCreq"),
+        Text(
+          extent={{-96,-54},{-72,-66}},
+          lineColor={0,0,255},
+          textString="SBHreq"),
+        Text(
+          extent={{62,62},{92,46}},
+          lineColor={217,67,180},
+          textString="yOcc"),
+        Text(
+          extent={{64,8},{94,-8}},
+          lineColor={217,67,180},
+          textString="ySBC"),
+        Text(
+          extent={{66,-40},{96,-56}},
+          lineColor={217,67,180},
+          textString="ySBH"),
+        Ellipse(extent={{-38,36},{38,-38}}, lineColor={179,151,128},
+          fillPattern=FillPattern.Sphere,
+          fillColor={179,151,128}),
+        Line(
+          points={{0,34},{0,20}},
+          color={179,151,128},
+          smooth=Smooth.Bezier),
+        Line(
+          points={{0,-22},{0,-36}},
+          color={179,151,128},
+          smooth=Smooth.Bezier),
+        Line(
+          points={{0,7},{0,-7}},
+          color={179,151,128},
+          smooth=Smooth.Bezier,
+          origin={-28,-1},
+          rotation=90),
+        Line(
+          points={{0,7},{0,-7}},
+          color={179,151,128},
+          smooth=Smooth.Bezier,
+          origin={28,1},
+          rotation=90),
+        Ellipse(extent={{-2,2},{2,-2}},   lineColor={179,151,128},
+          fillColor={162,29,33},
+          fillPattern=FillPattern.Solid),
+        Polygon(
+          points={{0,2},{24,34},{26,32},{2,0},{2,2},{0,2}},
+          lineColor={179,151,128},
+          smooth=Smooth.Bezier,
+          fillColor={162,29,33},
+          fillPattern=FillPattern.Solid)}),
+      Diagram(coordinateSystem(preserveAspectRatio=false, extent={{-120,-100},{140,
+            100}}), graphics={
+        Rectangle(
+          extent={{-120,98},{140,8}},
+          lineColor={179,151,128},
+          lineThickness=0.5,
+          fillPattern=FillPattern.Solid,
+          fillColor={215,215,215}),
+        Text(
+          extent={{70,92},{134,78}},
+          lineColor={0,0,0},
+          lineThickness=0.5,
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          textString="Pass through occupancy signal or force
```

Comment:
better to break the text string to multiple strings. It will be easier to adjust the font size.